### ISSUE TITLE
Style guide

### ISSUE THREAD

**Issue by Swepedia** [aryanhimanshu/Algorithms#165]
I feel like this repo needs a very basic style guide to resolve things like tabs vs. spaces, and what line curly braces go on for languages with that aspect. I don't think it could be more specific than that because of all the different languages being used. Unless someone wants to make specific rules for each language, but that would be a lot of work.

**Comment by LuckehPickle** [comment]
It definitely needs some kind of organisation as well. At the moment, some things are grouped by language, some things are grouped by algorithm, and sometimes both or neither. 

Perhaps it could be changed to something like: `algorithm/language`. For example, you could find a linear search in ruby under the directory: `Linear Search/Ruby/linear_search.rb`.

**Comment by Swepedia** [comment]
Agreed. I think we could leave it mostly the same, just add folders for the languages within each category (Data Structures, Search, and Sort). It would also help if the maintainer was more active. @aryanhimanshu are you there?
